src/Shared/Helpers/ConsoleHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.CST.OpenSource.Shared
+{
+    public class ConsoleHelper
+    {
+        static StreamWriter streamWriter;
+        static FileStream fileStream;
+
+        public static void RedirectConsole(string outFile)
```

Comment:
Should we instead return a bool if we succesfully swapped?  This currently just swallows any errors with the input without any indication back.

Reply:
the reason was that there is nothing much to do if we couldn't switch, but I can return a bool and emit an error message